sop-website/docs/Waveform-Data/waveform-file-format.mdx

```diff
@@ -16,9 +16,12 @@ All waveform data must adhere to the following directory structure to facilitate
 
 ```plaintext
 .
-├── OMOP
-|  └── waveform_visit_links.csv             # File linking visit occurrence and waveform session
-├── SUBMISSION.md  
+├── Metadata
```

Comment:
lower case for consistency with other folder names?

Reply:
@tompollard thanks for the feedback. I use "Metadata" for consistency with what the upload tool expects: https://github.com/chorus-ai/chorus-extract-upload?tab=readme-ov-file#configuration . However, I see that it also expects "Waveforms" instead of "waveforms". I assume our tools are flexible enough that this won't usually matter but I'll check with @tcpan regarding whether the ingestion tool is case insensitive. We can then be consistent one way or the other with the Waveform SOP.

Reply:
We discussed this in the Waveform meeting today with @tcpan and the data acquisition team. We agreed to stick with capitalizing the first letter in the folder names which is consistent with other SOPs. I've updated this SOP to be consistent with that.